test/commands/commit/get.test.ts

```diff
@@ -17,10 +17,8 @@ describe("commands/commit/get", () => {
   describe("given a commit ID arg, and no flags", () => {
     test
       .env({ KNOCK_SERVICE_TOKEN: "valid-token" })
-      .stub(
-        KnockApiV1.prototype,
-        "getCommit",
-        sinon.stub().resolves(
+      .stub(KnockApiV1.prototype, "getCommit", (stub) =>
+        stub.resolves(
```

Comment:
This was another breaking change from the oclif migration guide: https://github.com/oclif/fancy-test?tab=readme-ov-file#v3-breaking-changes

The oclif test helper lib now uses `sinon` under the hood for stubbing, and it's now expected to use the provided stub object instead of calling `sinon.stub()` directly.